src/main/kotlin/com/terraformation/backend/support/SupportService.kt

```diff
@@ -59,11 +61,15 @@ class SupportService(
       filename: String,
       sizedInputStream: SizedInputStream,
   ): List<TemporaryAttachmentModel> {
+    val tikaContentType = MediaType.parseMediaType(Tika().detect(sizedInputStream))
     sizedInputStream.contentType?.let {
       !isContentTypeSupported(it) &&
           throw NotSupportedException(
               "$it is not a supported content type. Must be one of $SUPPORTED_CONTENT_TYPES_STRING")
     }
+    !isContentTypeSupported(tikaContentType) &&
+        throw NotSupportedException(
+            "File detected to be $tikaContentType, which is not supported. Must be one of $SUPPORTED_CONTENT_TYPES_STRING")
```

Comment:
Didn't notice this in the existing code when I reviewed it, but I think this, and the one above it, would be clearer as `if` statements than as short-circuit boolean expressions.